.circleci/config.yml

```diff
@@ -309,7 +324,8 @@ commands:
             ng lint ddp-sdk
             ng lint toolkit
             ng lint $ANGULAR_PROJECT_NAME
-  run-tests:
+
+  run-ng-test:
```

Comment:
Rename command `run-tests` to `run-ng-test` to make it clear. Not to confuse with e2e tests.